### PR TITLE
Fix many depreciation warnings

### PR DESCRIPTION
**Pull request by dgilman**
Note that this likely bumps to a minimum requirement of SQLAlchemy 1.4+

### CODE REVIEW

File: flask_admin/contrib/sqla/ajax.py [hunk 1]
```diff
@@ -58,16 +58,13 @@ def format(self, model):
 
         return getattr(model, self.pk), as_unicode(model)
 
-    def get_query(self):
```
Comment (mrjoes) [inline flask_admin/contrib/sqla/ajax.py:61]: This is incompatible change. `get_query` allowed subclassing and mutating the generic query object in child admin classes.

**Comment by mrjoes** [comment]
Unfortunately can't accept this one in current form - it changes Flask-Admin SQLa-related APIs.

**Review by ankit-gautam23 — APPROVED** [review]
LGTM

**Comment by alanhamlett** [comment]
Closing for now. Will re-open if `get_query` api made backwards compatible.

**Pull request closed without merging**